cloud-sql/r2dbc/pom.xml

```diff
@@ -36,21 +36,14 @@
       <artifactId>spring-boot-starter-thymeleaf</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.springframework.data</groupId>
-      <artifactId>spring-data-r2dbc</artifactId>
-      <version>1.2.3</version>
-    </dependency>
-    <dependency>
-      <groupId>io.r2dbc</groupId>
-      <artifactId>r2dbc-pool</artifactId>
-      <version>0.8.5.RELEASE</version>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-r2dbc</artifactId>
```

Comment:
should we have a version here for `spring-boot-starter-data-r2dbc`?

Reply:
Spring Boot provides dependency management for Spring Data artifacts, so unless there is a specific reason to deviate from the version provided by Spring Boot it is recommended to not manage that version yourself in your project.

Reply:
sounds good.